addOns/openapi/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
  - Fixed var support in URLs ([Issue #6726](https://github.com/zaproxy/zaproxy/issues/6726))
  - Import file definition even if it has issues ([Issue #6758](https://github.com/zaproxy/zaproxy/issues/6758)).
+- The import progress is now displayed using a Progress Panel.
```

Comment:
Could be in Added instead (or Changed).

Reply:
Okay, mind if I also tweak the spaces leading the other bullets?

Reply:
Not at all.